docs/contribute/processes/development.md

```diff
@@ -2,13 +2,41 @@
 title: Development Process
 ---
 
-Our development workflow bases on the standard [GitHub workflow](https://guides.github.com/introduction/flow/) and consists of the following steps:
+Our development workflow bases on the standard [GitHub workflow](https://guides.github.com/introduction/flow/) with the slight variation that developers are working on forks of the main repository.
+The workflow consists of the following steps:
 
+0. [Fork](https://guides.github.com/activities/forking/) the main Saros repository. If you already have a fork of the repository, you can skip this step.
 1. [Create a remote branch](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/) with a meaningful name.
 2. Push changes to the new branch.
-3. Create a [pull request](https://help.github.com/articles/about-pull-requests/). Don't forget to invite reviewers!
-4. Improve the pull request (or explain your changes) until the reviewers are satisfied (see our [review process](review.md) for more information).
-5. [Squash and merge](https://help.github.com/articles/about-pull-request-merges/#squash-and-merge-your-pull-request-commits) pull request if:
-    * Reviewers approved the pull request.
-    * The tests in the CI server are successful.
-    
+3. Create a [pull request](https://guides.github.com/activities/forking/#making-a-pull-request) onto the main repository. General information about pull requests can be found [here](https://help.github.com/articles/about-pull-requests/).
+    * *(Optional)* Request other developers to review the pull request by asking them directly through [mentions](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#mentioning-people-and-teams).
+4. Improve the pull request (or explain your changes) until the reviewers are satisfied (see our [review process](review.md) for more information) and the [CI](https://www.saros-project.org/contribute/processes/continuous-integration.html) jobs run successfully.
+5. Request that the pull request is merged into the main repository if
+    * The CI builds are successful.
+    * The pull request was approved by at least some of the reviewers.
+    * There are no open requests for changes by a reviewer.
+
+## Pull Request Structure
```

Comment:
Yes, the developers won't be able to execute the merge themselves. (Which is why the steps above state that they should request a core developer to merge the PR.)

But, as different merge strategies require the PRs to be structured in different ways, I think it is still useful to explain this to the developers. This (hopefully) allows them to structure their PRs accordingly so that we can merge them easily and don't have to have discussions about splitting commits, cleaning up the history, etc.